DOCKERFILE/docker-compose.yml

```
@@ -3,7 +3,7 @@ version: '3.8'
 services:
   webrtc-pi:
     build:
-      context: ./server
+      context: .
```

컨테이너 간 통신을 염두에 두고 각 컨테이너에 필요한 소스 파일과 Dockerfile을 분리하여 디렉토리 구조를 설계하는 것은 좋을 것 같습니다.

```suggestion
project-root/
├── docker-compose.yml         
├── server/                  
│   ├── Dockerfile
│   └── src                   
└── Ai_model_1/              
    ├── Dockerfile
    └── model/               
        └── model.py

```